mucyomiller/PropertyPro-Lite
# #166862845 ui improvements

#### What does this PR do?
adds UI improvements
#### Description of Task to be completed?
as part of UI improvements, it adds custom google map marker (red-flag) & changes lock icon (sign out) of material design icons to font awesome's one. 
#### How should this be manually tested?
1) navigate to the details page of property and view custom google map  marker    
2) navigate to add advert page of the user(agent)  after you log in and see fa lock icon
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#166862845
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
